synapse/lib/config.py

```diff
@@ -178,14 +263,20 @@ def setConfOpts(self, opts):
         '''
         Use settings from the given dict to update the object config.
 
-        Example:
+        Args:
+            opts (dict):
+
+        Examples:
+            Set a pair of keys on a object using a dictionary::
 
-            opts = {
-                'foo:enabled':True,
-                'foo:size':1000
-            }
+                opts = {
+                    'foo:enabled':True,
```

Comment:
should there be a space between : and True? (not sure if style should be enforced in docstrings)

Reply:
yeah - we should be good citizens there.